GUI/src/cz/fidentis/gui/comparison_batch/BatchRegistrationConfiguration.java

```diff
@@ -1071,11 +1098,11 @@ public void run() {
                             m.setVerts(procrustes.getGpa().getPA(i).getVertices());
                             procrustes.getGpa().getPA(i).updateFacialPoints(c.getFacialPoints(m.getName()));                            
                             
-                            
-                            if (m.getName().equals(tc.getViewerPanel_Batch().getListener().getModel().getName())) {
+                            //Update canvas with registered model
+                            /*if (m.getName().equals(tc.getViewerPanel_Batch().getListener().getModel().getName())) {
```

Comment:
Jop, myslienka, ak si spravne pamatam, bola taka, ze po tom, co sa zarovnaju tvare pomocou landmarkov, tak sa posunu tvare a posunu sa landmarky, takze by sa mala po vypocte updatnut pozcia landmarkov (tvar sa uz updatuje), ale v batchi sa deje taka prazvlastna vec, po tom co sa vsetko vypocita a zvola sa metoda UpdateNavigator() -- alebo cosi take -- tak sa updatnuta tvar vrati na povodnu nacitanu (nezarovnanu) tvar, takze som to zakomentovala, aby sa neupdatovala pozicia landmarkov a nechala to tam, aby sa to riesilo potom eventualne :)